mpqp/core/circuit.py

```diff
@@ -265,7 +265,7 @@ def hard_copy(self):
             c: 4/══════╩══╩═════╩══╩═
                        0  1     2  3
             NoiseModel:
-                Depolarizing(0.01, [all], 1)
+                Depolarizing(0.01)
             >>> circuit2.nb_qubits = 3
```

Comment:
I just noticed that the example given here is an example on how some instruction have a dynamic size, and is not at all about the method (it is not even called in the example!), this should be fixed

Reply:
For repr it is '[]', and for now, there is no str.